db/schema.rb

```diff
@@ -16,7 +16,6 @@
   enable_extension "plpgsql"
 
   create_table "questions", force: :cascade do |t|
-    t.integer "chart_id"
```

Comment:
This schema is auto generated and should not be changed without the autogenerated migration. Each time the migrations are run they'd just overwrite this.